tests/dragonfly/replication_test.py

```diff
@@ -1246,7 +1246,10 @@ async def block_during_takeover():
         "Add a blocking command during takeover to make sure it doesn't block it."
         start = time.time()
         # The command should just be canceled
-        assert await c_blocking.execute_command("BLPOP BLOCKING_KEY1 BLOCKING_KEY2 100") is None
+        try:
+            assert await c_blocking.execute_command("BLPOP BLOCKING_KEY1 BLOCKING_KEY2 100") is None
+        except redis.exceptions.ConnectionError as e:
```

Comment:
but why do we have connection error?
is it because takeover already started? if so isnt the solution is to increase a little bit the delay in takeover execution ?

Reply:
> if so isnt the solution is to increase a little bit the delay 

Or just handle the ConnectionError. It doesn't eve happen that often :man_shrugging:

Reply:
The problem with this kind of a fix that now you see it does not happen that often, but tomorrow someone will change the test and will remove the delay in takeover or something else in the test will change and we will not know this functionality is not tested at all as it might always get to the path of ConnectionError